openquake/sep/classes.py

```diff
@@ -227,7 +227,7 @@ class ZhuEtAl2017LiquefactionCoastal(SecondaryPeril):
     to binary output via the predefined probability threshold.
     """
 
-    outputs = ["LiqProb", "LiqOccur", "LSE"]
+    outputs = ["LiqProb", "LiqOccur", "LqSE"]
```

Comment:
If we are using Lq for liquefaction, can we also change LiqProb and LiqOccur to LqProb and LqOccur to be consistent?